python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -375,7 +389,7 @@ def warpStamps(self, stamps, pixCenters):
         nb90Rots = np.argmin(np.abs(possibleRots - float(yaw)))
 
         # apply transformation to each star
-        warpedStars = []
+        warpedStars, warpTransforms, XY0s = [], [], []
```

Comment:
I think this should be `xy0s`, since we only capitalize class names, and the use of `xy0` is common in the stack.